Xwt/Xwt.Backends/BackendHost.cs

```diff
@@ -171,12 +173,15 @@ protected void LoadBackend ()
 			}
 		}
 
+		List<object> enabledEvents = new List<object> ();
```

Comment:
Wouldn't it better to use `HashSet`?

Reply:
Could be.

Reply:
Actually, since we don't have to check if events are registered, a list is better.